README.md

```diff
@@ -0,0 +1,22 @@
+#SCHOOL DIARY
+###Project description
+###Configuration
+
```

Comment:
I think these configuration should be used in application.properties files - not in a run configuration in IntelliJ